packages/SwingSet/src/controller/initializeSwingset.js

```diff
@@ -244,7 +244,10 @@ export function swingsetIsInitialized(hostStorage) {
  * @param {SwingSetConfig} config
  * @param {string[]} argv
  * @param {HostStore} hostStorage
- * @param {{ kernelBundles?: Record<string, string>, verbose?: boolean }} initializationOptions
+ * @param {{ kernelBundles?: Record<string, string>, verbose?: boolean,
+ *           addVatAdmin?: boolean, addComms?: boolean, addVattp?: boolean,
+ *           addTimer?: boolean,
+ *         }} initializationOptions
```

Comment:
This type seems brittle.  Maybe at least define it by name somewhere?

Reply:
yeah, I suspect we have multiple definitions for this particular thing, that need to be consolidated, but I'll poke around and see if defining `InitializationOptions` is an option